### DISCUSSION TITLE
Expected release cadence for Docfx

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/docfx#8471] [General]
At the time of writing, it is 3 weeks since a release was made.  Appreciating that the work on Docfx may be done by volunteers or people with other demands on their time, would it nevertheless be possible to set some expectations on release cadence?  

Docfx is an increasingly important component in many project's build pipelines but it currently seems hard to know whether it can be relied upon if fixes are needed.  Many thanks.

**Comment by yufeih** [comment]
Usually there is a release if major features or important bug fixes land. I don't have as much personal time for this project in the last couple of weeks as in holiday seasons last December or Jan, nevertheless the next release would probably come with a refreshed site template soon.

Small community fixes can be reviewed, merged and released in matter of days, provided they are in good quality and come with tests. Is there a particular bug fix that you are looking for in the main branch to be released?

**Reply by ghost** [reply]
Thanks Yufei - I was after the fixes that were done for the "nested" layout  (https://github.com/dotnet/docfx/pull/8424).  I tried getting the latest source from the main branch (as the PR was accepted some time ago) but struggled getting the build to work and don't have time to diagnose, so need to wait for the actual release.

The question was more general though, just to understand what to expect in terms of new releases.

**Reply by yufeih** [reply]
Thanks! Released https://github.com/dotnet/docfx/releases/tag/v2.62.2 that contains the nested layout fix.

**Reply by ghost** [reply]
Yufei - sorry, been offline for a week whilst travelling - many thanks for this, v much appreciated!
